### PR TITLE
RightsCon 2021 abstract

### PR DESCRIPTION
**Pull request by LauraCarter**
### Summary

Proposal to present a tech demo talk at RightsCon 2021 on the Ethics Book, per this issue https://github.com/alan-turing-institute/the-turing-way/issues/1717

### List of changes proposed in this PR (pull-request)

* Adds abstract to the conference folder for an 8-10 minute, pre-recorded demo
* Updates the conference readme to add links to RightsCon and the abstract

### What should a reviewer concentrate their feedback on?

- [ ] Suggest edits, changes, additions to the proposal - particularly ways to make the video reusable for the future
- [ ] Also chime in if you'd like to be involved with the video itself!

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: conferences/abstracts/2021-01-RightsCon.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Racial justice,
+Gender justice,
+Youth and children's rights,
+Migration)_
+No
+
+### Session title (in English)
+_20 words_ 
+_Pique the audience's interest while also explainining the 'big idea' of your session_
+The Turing Way: Making reproducible, ethical and inclusive data science too easy not to do
```
Comment (lauracion) [inline conferences/abstracts/2021-01-RightsCon.md:29]:
```suggestion
The Turing Way: Making reproducible, ethical, and inclusive data science too easy not to do
```

File: conferences/abstracts/2021-01-RightsCon.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+## Format and Goal
+
+### Session format (choose from drop-down)
+Tech Demo - A brief (8-10 minute), prepared presentation on a new technology, tool, or technical project. Demos are recorded in advance of the summit.
+
+## Session Concept 
+### What’s your product, initiative, or innovation?  (150 words)
+_In a few sentences, tell us what you plan to demo. (Yes, we know it’s not much space – make it count!)_
+The Turing Way is a book, a community and a global collaboration.
```
Comment (lauracion) [inline conferences/abstracts/2021-01-RightsCon.md:44]:
```suggestion
The Turing Way is a book, a community, and a global collaboration. 
```

File: conferences/abstracts/2021-01-RightsCon.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+## Format and Goal
+
+### Session format (choose from drop-down)
+Tech Demo - A brief (8-10 minute), prepared presentation on a new technology, tool, or technical project. Demos are recorded in advance of the summit.
+
+## Session Concept 
+### What’s your product, initiative, or innovation?  (150 words)
+_In a few sentences, tell us what you plan to demo. (Yes, we know it’s not much space – make it count!)_
+The Turing Way is a book, a community and a global collaboration. 
+This presentation will demonstrate one of the five open source volumes that form the project: the Guide to Ethical Research.
```
Comment (lauracion) [inline conferences/abstracts/2021-01-RightsCon.md:45]:
```suggestion
This presentation will demonstrate one of the five open-source, online, collaborative books that form the project: the Guide to Ethical Research.
```

File: conferences/abstracts/2021-01-RightsCon.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+(The other four guides cover Reproducible Research, Project Design, Communication, and Collaboration).
```
Comment (lauracion) [inline conferences/abstracts/2021-01-RightsCon.md:46]:
```suggestion
The other four guides cover Reproducible Research, Project Design, Communication, and Collaboration.
```

File: conferences/abstracts/2021-01-RightsCon.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+We will talk about the aims of the project, the community we are building, and most importantly - how you can get involved!
```
Comment (lauracion) [inline conferences/abstracts/2021-01-RightsCon.md:47]:
```suggestion
We will talk about the aims of the project, the community we are building, and, most importantly, how you can get involved!
```

File: conferences/abstracts/2021-01-RightsCon.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+### What makes you the right person to speak about this product, initiative, or innovation? (150 words)
+_Now’s the time to sell yourself as a speaker. Don’t hold back._
+
+Laura is a human rights researcher who is one of the Ethics Guide collaborators: as well as developing content that is practical and accessible for data researchers, she also guides new contributors as a new open science contributor herself!
+
+Malvika is the Community Manager of the project: based at the Alan Turing Institute in the UK, she brings her extensive experience in open science and collaboration to a global community spanning 6 continents.
```
Comment (lauracion) [inline conferences/abstracts/2021-01-RightsCon.md:54]:
```suggestion
Malvika is the Community Manager of The Turing Way. She is based at The Alan Turing Institute in the United Kingdom and brings her extensive experience in open science and collaboration to a global community spanning 6 continents. 
```

**Review by lauracion — APPROVED** [review]
It looks great, Laura. Only a few minor edits that won´t change much. Gook luck!

I am not sure if it can be done in 8 minutes, but having a video with the basics on how to address a "good first issue" in TTW may be a good and reusable video :-) As usual, I wish I could get more involved - not for now, sorry.

**Comment by LauraCarter** [comment]
Thanks @lauracion! I think focusing on how to address a good first issue is a really good idea :) if it gets accepted, I think we should see if we could fit that into 8-10 mins!

File: conferences/abstracts/2021-01-RightsCon.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+### Session format (choose from drop-down)
+Tech Demo - A brief (8-10 minute), prepared presentation on a new technology, tool, or technical project. Demos are recorded in advance of the summit.
+
+## Session Concept 
+### What’s your product, initiative, or innovation?  (150 words)
+_In a few sentences, tell us what you plan to demo. (Yes, we know it’s not much space – make it count!)_
+The Turing Way is a book, a community, and a global collaboration. 
+This presentation will demonstrate one of the five open-source, online, collaborative books that form the project: the Guide to Ethical Research.
+The other four guides cover Reproducible Research, Project Design, Communication, and Collaboration.
+We will talk about the aims of the project, the community we are building, and, most importantly, how you can get involved!
```
Comment (LauraCarter) [inline conferences/abstracts/2021-01-RightsCon.md:47]:
```suggestion
We will talk about the aims of the project, the community we are building, and, most importantly: how you can get involved!
```

**Review by malvikasharan — APPROVED** [review]
Looks great. I will do a bit formatting before you merge this!

File: conferences/abstracts/2021-01-RightsCon.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+Racial justice,
+Gender justice,
+Youth and children's rights,
+Migration)_
+No
+
+### Session title (in English)
+_20 words_ 
+_Pique the audience's interest while also explainining the 'big idea' of your session_
+The Turing Way: Making reproducible, ethical, and inclusive data science too easy not to do
```
Comment (LauraCarter) [inline conferences/abstracts/2021-01-RightsCon.md:29]:
```suggestion
The Turing Way: Building a collaborative guide to ethical data science
```

File: conferences/abstracts/2021-01-RightsCon.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+## Session Concept 
+
+### What’s your product, initiative, or innovation?  (150 words)
+_In a few sentences, tell us what you plan to demo. (Yes, we know it’s not much space – make it count!)_
+
+The Turing Way is a community, and a global collaboration.
+This presentation will demonstrate one of the five open-source, online, collaborative books that form the project: the Guide to Ethical Research.
+The other four guides cover Reproducible Research, Project Design, Communication, and Collaboration.
+We will talk about the aims of the project, the community we are building, and, most importantly: how you can get involved!
```
Comment (malvikasharan) [inline conferences/abstracts/2021-01-RightsCon.md:53]:
```suggestion
In data science, we make data-driven decisions throughout the lifecycle of their project . 
It is our duty to ensure that our work maintains a high level of moral integrity as well as the highest scientific standards, especially when it impacts people’s lives.
In this presentation, we will introduce The Turing Way, an open source book project that involves and supports its diverse community in developing and sharing resources that make data science reproducible, ethical, collaborative and inclusive. 
In particular, we will demonstrate its guide for ethical research, the community behind its development, and, most importantly, how RightsCon attendees can get involved. (101/150 words)
```

Comment (malvikasharan) [reply conferences/abstracts/2021-01-RightsCon.md:53]: We have a lot of space to explain the ethical guide - as in, would you like to mention the titles of the chapters you have planned?

File: conferences/README.md [hunk 10]
```diff
@@ -41,3 +41,4 @@ THANK YOU to every member of the community who spreads the word! :sparkling_hear
 | [FSCI2020 Lightening Talk](https://www.force11.org/fsci/2020) | 3 August 2020 | [https://doi.org/10.5281/zenodo.3979179](https://doi.org/10.5281/zenodo.3979179) | [https://doi.org/10.5281/zenodo.3979179](https://doi.org/10.5281/zenodo.3979179) | [https://doi.org/10.5281/zenodo.3979179](https://doi.org/10.5281/zenodo.3979179)) |
 | [CarpentryConnect @ Home 2020](https://2020.carpentrycon.org/) | 7 August 2020 | [2020-08-CarpentryCon-at-home-lightning.md](./abstracts/2020-08-CarpentryCon-at-home-lightning.md) | [TBA](https://zenodo.org/record/TBA) | [TBA](http://doi.org/10.5281/zenodo.TBA) |
 | [ACM FAccT Conference 2021](https://facctconference.org/2021/cfw.html) | 3-10 March 2021 | [https://zenodo.org/record/4420152#.X_S-VeAaT1w](https://zenodo.org/record/4420152#.X_S-VeAaT1w) | Awaiting acceptance/rejection
+| [RightsCon2021](https://www.rightscon.org/program/) | 7-11 June 2021 | [2021-01-RightsCon.md](./abstracts/2021-01-RightsCon.md) | Not yet submitted (deadline is 26 Jan 2021)
```
Comment (LauraCarter) [inline conferences/README.md:44]:
```suggestion
| [RightsCon2021](https://www.rightscon.org/program/) | 7-11 June 2021 | [2021-01-RightsCon.md](./abstracts/2021-01-RightsCon.md) | Submitted 25 Jan 2021
```

**Pull request merged**